### PR TITLE
Add keyboard navigation support for month view mode using arrow keys

### PR DESCRIPTION
**Pull request by goodmorninggoaway**
Improve accessibility of datepicker by adding keyboard support when using month view mode

### CODE REVIEW

File: css/datepicker3.css [hunk 1]
```diff
@@ -748,7 +750,7 @@ fieldset[disabled] .datepicker table tr td span.active.disabled:hover.active {
   min-width: 16px;
   padding: 4px 5px;
   font-weight: normal;
-  line-height: 1.428571429;
+  line-height: 1.42857143;
```
Comment (acrobat) [inline css/datepicker3.css]: Please revert this line

File: css/datepicker3.css [hunk 2]
```diff
@@ -782,7 +784,7 @@ fieldset[disabled] .datepicker table tr td span.active.disabled:hover.active {
   color: #333333;
   font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
   font-size: 13px;
-  line-height: 1.428571429;
+  line-height: 1.42857143;
```
Comment (acrobat) [inline css/datepicker3.css]: Please revert this line

**Comment by acrobat** [comment]
@goodmorninggoaway please fix the comments and add the style changes to the datepicker.less (bootstrap 2). When that's done I will do final review! Thanks!

**Comment by goodmorninggoaway** [comment]
@acrobat I fixed the comments. Seemed like a rounding issue from a local build.

**Comment by acrobat** [comment]
Great thanks @goodmorninggoaway! Can you just rebase the pr against the current master? Then should this PR ready for merge!

**Comment by acrobat** [comment]
ping @goodmorninggoaway

**Comment by acrobat** [comment]
@goodmorninggoaway Can you also add keyboard navigation for the year view? Fixes #1344

**Comment by goodmorninggoaway** [comment]
@acrobat Ok. I am going to decline this PR and submit a new one when complete.

**Comment by acrobat** [comment]
Ok! You can also continue work on this branch if you want to!

**Pull request closed without merging**